receiver/kafkareceiver/go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/open-telemetry/opentelemetry-collector-contrib/receiver/kafkareceiver
 
-go 1.23.0
+go 1.23.8
```

Comment:
we would like to use go 1.23.0 whenever applicable, see https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/40065